pkg/test/framework/components/istio/operator.go

```diff
@@ -249,7 +249,7 @@ func deploy(ctx resource.Context, env *kube.Environment, cfg Config) (Instance,
 		return nil, fmt.Errorf("%d errors occurred deploying remote clusters: %v", errs.Len(), errs.ErrorOrNil())
 	}
 
-	if env.IsMulticluster() {
+	if env.IsMulticluster() && !isCentralIstio(env, cfg) {
 		// For multicluster, configure direct access so each control plane can get endpoints from all
 		// API servers.
 		if err := configureDirectAPIServerAccess(ctx, env, cfg); err != nil {
```

Comment:
Maybe we don't need this here and we can always call `configureDirectAPIServiceAccessForCluster` when deploying remote clusters in `deployControlPlane`